testem.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable import/no-commonjs */
+/* eslint-disable flowtype/require-valid-file-annotation */
+const generateFixtureJson = require('./test/integration/lib/generate-fixture-json');
+const createServer = require('./test/integration/lib/server');
+const runAll = require('npm-run-all');
+
+let beforeHookInvoked = false;
+let afterHookInvoked = false;
+let server;
+
+module.exports =  {
+    "framework": "tap",
+    "src_files": [
+        "test/integration/dist/*.js"
+    ],
+    "serve_files": [
+        "test/integration/dist/tape.js",
+        "test/integration/dist/mapbox-gl-test.js",
```

Comment:
Remove this rollup target destination and just use `dist/mapbox-gl-dev.js` instead. It would allow a simpler dev workflow where the debug pages and the test pages can be open simulatneously.

Reply:
Done, you can now have both open simultaneously